pygments/lexers/bdd.py

```diff
@@ -0,0 +1,126 @@
+"""
+    pygments.lexers.bdd
+    ~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for BDD language.
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.lexer import RegexLexer, include, bygroups
+from pygments.token import Comment, Keyword, Name, String, Number, Generic, Text
+
+__all__ = ['BddLexer']
+
+class BddLexer(RegexLexer):
```

Comment:
Needs a `.. versionadded: 2.10` comment (and in general a comment explaining what this lexer is about.)

Reply:
Done. Thanks for your reminder 😃

Reply:
Done.

Reply:
Done.